src/internal/operations/backup.go

```diff
@@ -138,6 +138,16 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 		observe.Complete()
 	}()
 
+	var runnable bool
+
+	// IsBackupRunnable checks if the user has services enabled to run a backup.
+	// it also checks for conditions like mailbox full.
+	runnable, err = op.bp.IsBackupRunnable(ctx, op.Selectors.PathService(), op.ResourceOwner.ID())
+	if err != nil || !runnable {
```

Comment:
On a response of `false, nil` this is going to log and fail a nil error, which doesn't make much sense.

The two cases should be split into separate conditionals.